cypress/e2e/notes-100/access-shared-note.cy.js

```diff
@@ -1,16 +1,39 @@
 import {waitForModel, homepageSetup} from '../../support/utils'
 
-
-describe('select-a-note', () => {
-  context('Open the link with a shared note', () => {
+// From https://github.com/bldrs-ai/Share/issues/1072
+describe('Notes 100 - Access shared note', () => {
```

Comment:
- Use utils.js#setIsReturningUser instead of setCookie (setCookie semantics are hard to understand.. the 1 is a serial, not a bool, and will change). 

- Since you're doing both homepageSetup and setIsReturningUser, for all tests here, move them together under describe
```
describe('Notes 100 - Access shared note', () => {
  beforeEach(() => {
    homepageSetup()
    setIsReturningUser()
  })
```

Reply:
I see. Cool. Addressed.